docker/build/installers/py3_requirements.txt

```diff
@@ -42,3 +42,5 @@ h5py
 pyyaml
 utm
 
+#Github
+pygithub
```

Comment:
Since pygithub is only used for CI, we'd better not include it in Docker images.

Reply:
If not, CI will install it every time when container restart, which is very inefficient.